benchmark/opperf/README.md

```diff
@@ -72,6 +73,8 @@ python incubator-mxnet/benchmark/opperf/opperf.py --output-format json --output-
 
 3. **dtype** : By default, `float32`. You can override and set the global dtype for all operator benchmarks. Example: --dtype float64.
 
+4. **profiler** : By default, 'native'. You can override and set the global profiler for all operator benchmarks. Example: --profiler 'python'.
```

Comment:
What does `native` mean? Could you be more specific

Reply:
Native profiler uses the MXNet's default profiler function
Python uses python's time-it function

Reply:
I know what it means. Could you provide more information to first time users?

Reply:
Added a line. @apeforest